test/verify/check-login

```diff
@@ -408,6 +408,7 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
     @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804", "ubuntu-2004")
     @skipImage("No semanage", "fedora-coreos")
     def testSELinuxRestrictedUser(self):
+        return
```

Comment:
.. and of course that needs to go :-) I can take a look tomorrow morning, or is that just about ignoring the additional error message?

Reply:
Out of all the remaining issues, this is the one that I'd more appreciate your support on.

Reply:
I'd suggest to just do this for now:
```diff
commit 2fb1e38657630ef594d339a5e2db98fa20c5dbb9 (HEAD -> login-messages)
Author: Martin Pitt <mpitt@redhat.com>
Date:   Fri Mar 13 14:53:57 2020 +0100

    FIXUP: unbreak stuff
    
    Do run the test case, just ignore the message. It's ok if we can't pass
    that information to a restricted user session.

diff --git test/verify/check-login test/verify/check-login
index 8aa5335e3..65c748e9b 100755
--- test/verify/check-login
+++ test/verify/check-login
@@ -408,7 +408,6 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
     @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804", "ubuntu-2004")
     @skipImage("No semanage", "fedora-coreos")
     def testSELinuxRestrictedUser(self):
-        return
         m = self.machine
         b = self.browser
 
@@ -429,6 +428,8 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         self.allow_journal_messages(".*/var/lib/cockpit/machines.json.*Permission denied")
         self.allow_journal_messages('.*type=1400.*avc:  denied  { map }.*comm="cockpit-pcp".*')
         self.allow_journal_messages('.*type=1400.*avc:  denied .* comm="sudo".*')
+        # SELinux prevents passing memfd for restricted users
+        self.allow_journal_messages('Could not query seals on fd .*: not memfd.*')
         if m.image in ["fedora-30", "centos-8-stream"]:
             # older releases have more noise
             self.allow_journal_messages("sudo: .*setresuid.*: Operation not permitted")

```
This is enough to make it pass. I'll still look into finding the bit of the SELinux policy that damages the fd in the first place, then we can at least refer to either a bugzilla or (if it's deliberate), a rule.